src/Components/ProvisioningWizard/index.js

```diff
@@ -20,6 +21,10 @@ const ProvisioningWizard = ({ isOpen, onClose, image, ...props }) => {
   const [isConfirming, setConfirming] = React.useState(false);
   const [successfulLaunch, setLaunchSuccess] = React.useState();
 
+  const { isLoading, error: sourcesError, sources: availableSources } = sourcesForImage(image, { refetch: stepIdReached <= 2 });
+
+  if (isLoading) return null;
```

Comment:
This is a shortcut, it would probably be better to open the empty stater right away with spinner.
Should get done before merge IMO

Reply:
Fixed now